pallets/subtensor/tests/serving.rs

```diff
@@ -827,3 +827,136 @@ fn test_migrate_set_hotkey_identities() {
         );
     });
 }
+
+#[test]
+fn test_coldkey_swap_delegate_identity_updated() {
```

Comment:
This test should be in coldkey swap, and also extend the main coldkey swap test to check that the identity is swapped